docs/user/running-cincinnati.md

```diff
@@ -16,6 +16,34 @@ The Cincinnati deploymentconfig is available [here](../../dist/openshift/cincinn
 oc new-project cincinnati
 ```
 
+#### Create secret for container registry
+
+If Cincinnati needs credentials to access a container registry to fetch the release payload information , you need to create a secret with name `cincinnati-credentials`. Make sure the auth for the container registry is present in the config.json file.
+
+```shell
+oc create secret generic cincinnati-credentials --from-file=.dockerconfigjson=/home/lmohanty/.docker/config.json --type=kubernetes.io/dockerconfigjson
+```
+
+An example config.json
+
+```
+$ cat ~/.docker/config.json
+{
+        "auths": {
+                "https://index.docker.io/v1/": {
```

Comment:
URIs like this are legal, but they are not recommended.  Can we link to an auth-format spec like [this][1], and then just list a minimal example for pulling from a single registry?

[1]: https://github.com/containers/image/blob/v5.5.2/docs/containers-auth.json.5.md

Reply:
+1 to a documentation link. Release payloads are not mirrored to DockerHub so we should remove this section so that we wouldn't confuse users

Reply:
Fixed in https://github.com/openshift/cincinnati/pull/296/commits/57b9b01ed74d0a80299ea81a2d01047f1b6b6bc8